packages/site-kit/src/lib/markdown/renderer.js

```diff
@@ -754,9 +765,21 @@ async function find_nearest_node_modules(start_path) {
 async function create_snippet_cache(should) {
 	const snippet_cache = (await find_nearest_node_modules(import.meta.url)) + '/.snippets';
 
-	try {
-		if (should) await mkdir(snippet_cache, { recursive: true });
-	} catch {}
+	// No local cache exists yet
+	if (!CACHE_MAP.size) {
+		try {
+			if (should) await mkdir(snippet_cache, { recursive: true });
```

Comment:
The `should` should be on the outer `if(!CACHE_MAP.size)`

Reply:
Done